sources/openwrt-http.go

```diff
@@ -81,8 +81,13 @@ func (s *openwrt) Run() error {
 	} else {
 		switch s.definition.Image.ArchitectureMapped {
 		case "x86_64":
-			fname = fmt.Sprintf("openwrt-%s%s-generic-rootfs.tar.gz", releaseInFilename,
-				strings.Replace(architecturePath, "/", "-", 1))
+			if strings.Contains(release, "21.02.0-rc") {
```

Comment:
Is this specific to the release candidate or are you expecting this to continue to be true with final 21.02?
If the latter, then it should probably be changed to something like `strings.HasPrefix(release, "21.02")` or maybe @monstermunchkin has a better idea of how to handle this without needing to hardcode versions

Reply:
I don't know if the final will have n`-generic` as part of the filename of the rootfs. The release candidates have the same naming scheme as the Snapshots. 

We could make it check the prefix, and hope they keep the naming scheme :)